client/my-sites/domains/domain-management/name-servers/dns-templates.jsx

```diff
@@ -59,7 +59,7 @@ class DnsTemplates extends React.Component {
 					dnsTemplateService: dnsTemplates.MICROSOFT_OFFICE365.SERVICE,
 					modifyVariables: variables =>
 						Object.assign( {}, variables, {
-							mxdata: replace( variables.domain, '.', '-' ) + '.mail.protection.outlook.com.',
```

Comment:
The trailing dot is now applied automatically by Domain Connect. This is due to a change that is being made to the specification.